microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -283,7 +297,7 @@ When WebView2 has focus, it receives input directly from the user. An app may wa
 <!-- ---------- -->
 ###### Default background color
 
-WebView2 can specify a default background color. This can be any opaque color or transparent color. This color will be used if the webpage doesn't set its own background color.
+WebView2 can specify a default background color. This can be any opaque color or transparent color. This color will be used if the webpage doesn't set a background color or sets a transparent background color.
```

Comment:
```suggestion
WebView2 can specify a default background color. This can be any opaque color or transparent color. This color will be used if the webpage doesn't set a background color, or sets a transparent background color.
```